### PR TITLE
feat(ci/cd): added RT

### PR DESCRIPTION
**Pull request by paologallinaharbur**
- [x] The branch of the PR is prefixed with renovate/*
- [x] Tests pass in renovate branches
- [x] security workflow is included and will pass in renovate branches
- [x] the changelog is validated in push_pr workflow
- [x] The repository has the secrets available
- [x] Release schedule is documented and not colliding
- [x] rt-update-changelog is in Makefile
- [x] Tags with leading v are safe
- [x] Common renovate config is pointed
- [x] Potential troublesome dependencies are checked
- [x] pre-release and release workflows are updated to notify failures
- [x] CHANGELOG.partial.md is in .gitignore
- [x] release-toolkit action excluded/included fields are set
- [x] Docker image pins go patch version
- [ ] The dependency dashboard is sanitized --> It will be enabled when the PR is merged, and then we need to remove Worktato comment and set `dependencies` label.

### CODE REVIEW

File: .github/workflows/prerelease.yml
Comment (sigilioso) [inline .github/workflows/prerelease.yml:1]: Nit: why not rename files to unify? `prerelease.yml -> on_prerelease.yml`

**Review by sigilioso — APPROVED** [review]
Thanks! We may rename the files but I don't think it is important.

File: .github/workflows/prerelease.yml
Comment (paologallinaharbur) [reply .github/workflows/prerelease.yml:1]: True! I forget about that

File: go.mod [hunk 1]
```diff
@@ -19,7 +19,6 @@ require (
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/stretchr/objx v0.5.0 // indirect
```
Comment (sigilioso) [inline go.mod:22]: is this the result of `go mod tidy`? Should we add it to the check list?

Comment (paologallinaharbur) [reply go.mod:22]: I mean.. if we are touching all repos already it could make sense

**Pull request merged**